frontend/scss/app/components/_table.scss

```
@@ -111,3 +111,58 @@
+
+  }
+
+  &__paging-info {
+    display: flex;
+    flex-flow: row nowrap;
+    gap: var(--gap-larger);
+    align-items: center;
+  }
+
+  &__info {
+  }
```

Stylelint (17.10.0)

[error] Empty block (block-no-empty)

(block-no-empty)